docker/bin/upload-staticfiles.sh

```diff
@@ -24,11 +23,13 @@ docker/bin/move_hashed_staticfiles.py "${TMP_DIR}" "${TMP_DIR_HASHED}"
 for BUCKET in stage prod; do
     # hashed filenames
     aws s3 sync \
+        --no-progress \
```

Comment:
Doesn't this need `--quiet` and `--only-show-errors` in order to be applied?

Reply:
I think I may have missed that copying+pasting when being very careful to not accidentally deploy to the prod bucket when testing, I'll recheck.

Reply:
Ah, right, `--no-progress` does work alone ("This flag is only applied when the quiet and only-show-errors flags are *not* provided.") - however what it disables is the "dynamic" progress bar which horrifically messes with circle's console output, rather than listing each file which got uploaded.

I think that's still rather a lot of unnecessary output, so I'll change the command to `--only-show-errors`.